debian/rules

```diff
@@ -22,6 +22,19 @@ KERNEL_TYPE.esx := generic
 KERNEL_TYPE.gcp := gcp
 KERNEL_TYPE.kvm := kvm
 
+#
+# The DELPHIX_SIGNATURE_VERSION field refers to the current Delphix
+# Release version being built and only defaults to 6.0 if not set.
+# This is passed as an environment variable via jenkins job(s) that
+# launch the linux-pkg build for this package.
+#
+DELPHIX_SIGNATURE_VERSION := $(or $(DELPHIX_SIGNATURE_VERSION),6.0)
+
+#
+# Signing key types
```

Comment:
IMO, this comment should be removed, since it isn't providing any more value than what the code itself already provides. Alternatively, we could leave a comment that describes why we have these two key types, and perhaps what their intended usage is.

Reply:
Yeah, I'll remove the comment.

Reply:
Fixed.